eclipse/che
# Remove workspace loader from Che Server

### What does this PR do?
This PR deprecates workspace-loader since new dashboard has its own page for opening workspace and workspace loader is unused.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
It's part of https://github.com/eclipse/che/issues/18785

### How to test this PR?
No special test scenario, just build Che Server, run it and make sure workspaces work as expected.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
